src/components/header/header.module.css

```diff
@@ -137,17 +145,21 @@
 }
 
 @media screen and (min-width: 768px) {
-  .logoLink svg.greenwood-logo-full {
-    width: 60%;
+  .navBar {
+    display: flex;
```

Comment:
I _think_ we might want keep it as `display: contents` at this break point as well, as the vertical alignment of the navigation looks slightly better aligned vertically with the logo and mobile menu icon (IMO 😅 )

#### `display: flex` (current)
![Screenshot 2024-10-16 at 9 38 26 PM](https://github.com/user-attachments/assets/026c3cca-44eb-4ec4-8107-fc6c8ec95249)

#### `display: contents`
![Screenshot 2024-10-16 at 9 39 30 PM](https://github.com/user-attachments/assets/0ea2baad-88f6-4374-8cd2-c26868360191)

Reply:
Great catch.

Reply:
Hmm, now the nav links stay in the center as we get past tablet breakpoints, as opposed to staying to the right, against the side nav
![Screenshot 2024-10-17 at 11 30 14 AM](https://github.com/user-attachments/assets/5e4cd3af-3b66-4f2d-8259-6f6cf063d97b)

Apologies for not catching that, I am admittedly not super well-versed with flex box.  😬